src/pages/HomePage/CreateAppModal.js

```diff
@@ -54,17 +56,17 @@ class CreateAppModal extends Component {
 		});
 	};
 
-  componentWillReceiveProps = (nextProps) => {
-    const { createdApp } = nextProps;
-    const { hasJSON, appName } = this.state;
-    if (createdApp.data && createdApp.data.id) {
-      if (hasJSON) {
-        window.location.href = `app/${appName}/import`;
-      } else {
-        window.location.href = `app/${appName}`;
-      }
-    }
-  }
+	componentWillReceiveProps = (nextProps) => {
```

Comment:
Please don't use componentWillReceiveProps. You should use new lifecycle methods instead.